### ISSUE TITLE
Frequent flakes after moving to internal circle.

### ISSUE THREAD

**Issue by hsaraogi** [palantir/atlasdb#2809]
eg: build 8384.
```
cd "$CIRCLE_PROJECT_REPONAME"
./scripts/circle-ci/install-docker.sh
++ docker version --format '{{.Client.Version}}'
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
+ CURRENT_VERSION=1.10.3
+ [[ true == true ]]
+ MIN_VERSION_WANTED=1.13.1
++ echo -e '1.10.3\n1.13.1'
++ sort -V
++ head -n 1
+ '[' 1.13.1 '!=' 1.10.3 ']'
+ [[ true == true ]]
+ curl -fsSLO https://get.docker.com/builds/Linux/x86_64/docker-1.13.1.tgz
curl: (28) SSL connection timeout

cd "$CIRCLE_PROJECT_REPONAME"
./scripts/circle-ci/install-docker.sh
 returned exit code 28

Action failed: ./scripts/circle-ci/install-docker.sh
```

**Comment by hsaraogi** [comment]
We should enable external circle temporarily until we fix these flakes. Document the failures encountered in a centralized doc.

**Comment by hsaraogi** [comment]
Closing as almost fixed by #2843.
